src/framework/execution/__tests__/executeIntegration.test.ts

```diff
@@ -0,0 +1,18 @@
+import { IntegrationExecutionContext } from '../types';
+import { executeIntegrationLocally } from '../executeIntegration';
+import { LOCAL_INTEGRATION_INSTANCE } from '../instance';
+
+test('should execute validator function if provided in config', async () => {
+  const validate = jest.fn();
+
+  await executeIntegrationLocally({
+    invocationValidator: validate,
```

Comment:
I've wanted to rename this to `validateInvocation`, since it is a function that does something, not an object with properties (that is, verb vs. noun). What do you think?

Reply:
I'm cool with that name.